conf/battle/skill.conf

```diff
@@ -125,12 +125,15 @@ gvg_traps_target_all: 1
 // Default on official servers: 0 for Pre-renewal, 2 for Renewal
 //traps_setting: 0
 
-// Restrictions applied to the Alchemist's Summon Flora skill (add as necessary)
+// Restrictions applied to the Alchemist's Summon skills (add as necessary)
 // 1: Enable players to damage the floras outside of versus grounds.
-// 2: Disable having different types out at the same time
+// 2: Enable players to damage marine spheres outside of versus grounds.
+// 4: Disable having different types out at the same time
```

Comment:
This is really bad.
Please switch 2 and 4.
Otherwise this affects existing configurations.

Reply:
It affects existing configurations anyway since the setting has a new name. This order of settings is more logical.
And it should affect existing configurations since it's currently defaulting incorrectly for renewal.